packages/flutter/lib/src/material/material_button.dart

```diff
@@ -341,13 +342,24 @@ class MaterialButton extends StatelessWidget {
   /// Defaults to the value from the current [ButtonTheme].
   final double height;
 
+  /// Whether detected gestures should provide acoustic and/or haptic feedback.
+  ///
+  /// For example, on Android a tap will produce a clicking sound and a
+  /// long-press will produce a short vibration, when feedback is enabled.
```

Comment:
Same comment regarding long-press

Reply:
Removed comment in doc about long-press

Reply:
Commenting here that we've re-added the comment since long-press is now enabled on these buttons